src/cmake/stir_dirs.cmake

```diff
@@ -102,4 +102,5 @@ SET( STIR_TEST_DIRS
      test 
      test/numerics
      test/modelling
+     ${STIR_NIFTYPET_PROJECTOR_test}
```

Comment:
somewhat hacky. I'd rather say `if(NiftyPET_FOUND) ...` or whatever bool you're using